### PR TITLE
Add utcOffset to getDeviceTime result

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
I've changed the signature of the returned result, but this is fine, since this method is not invoked anywhere yet

### CODE REVIEW

File: lib/utilities.js [hunk 1]
```diff
@@ -41,13 +42,30 @@ async function getDeviceName (udid, socket) {
...
   const lockdown = await startLockdownSession(udid, socket);
   try {
-    const epochValue = await lockdown.getValue(LOCKDOWN_REQUEST.DEVICE_TIME);
-    const date = new Date(0); // The 0 there is the key, which sets the date to the epoch
-    date.setUTCSeconds(epochValue);
-    return date;
+    const timestamp = await lockdown.getValue(LOCKDOWN_REQUEST.DEVICE_TIME);
+    const utcOffset = await lockdown.getValue(LOCKDOWN_REQUEST.DEVICE_UTC_OFFSET);
+    return {
+      timestamp,
```
Comment (umutuzgur) [inline lib/utilities.js]: Don't you just wanna return a Date since it is the time object in JS

File: lib/utilities.js [hunk 2]
```diff
@@ -41,13 +42,30 @@ async function getDeviceName (udid, socket) {
...
+ */
 async function getDeviceTime (udid, socket) {
   const lockdown = await startLockdownSession(udid, socket);
   try {
-    const epochValue = await lockdown.getValue(LOCKDOWN_REQUEST.DEVICE_TIME);
-    const date = new Date(0); // The 0 there is the key, which sets the date to the epoch
-    date.setUTCSeconds(epochValue);
-    return date;
+    const timestamp = await lockdown.getValue(LOCKDOWN_REQUEST.DEVICE_TIME);
+    const utcOffset = await lockdown.getValue(LOCKDOWN_REQUEST.DEVICE_UTC_OFFSET);
```
Comment (umutuzgur) [inline lib/utilities.js]: Have you tested this? I think this returns a double

Comment (mykola-mokhnach) [reply lib/utilities.js]: the client will be using moment.js. I don't want to perform double transformation

File: lib/utilities.js (hunk 1 above)
Comment (mykola-mokhnach) [reply lib/utilities.js]: the client will be using moment.js. I don't want to perform double transformation

File: lib/utilities.js (hunk 2 above)
Comment (mykola-mokhnach) [reply lib/utilities.js]: this is fine

**Pull request merged**